uptick/assetlist.json

```diff
@@ -19,9 +19,10 @@
       "display": "uptick",
       "symbol": "UPTICK",
       "logo_URIs": {
-        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/uptick/images/uptick.png"
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/uptick/images/uptick.png",
```

Comment:
another PR has added these images, so please remove the SVG. Although we'd like to keep the reference to the SVG you've added here.